ratenyu/courses/templates/courses/detail.html

```diff
@@ -38,13 +38,14 @@ <h2>{{ reviews_list|length }} Reviews</h2>
     <ul class="detail-side-links">
         <li class="detail-side-links"><span class="rev-title">Professor:</span></li>
         {% for professor in professors_list %}
-        <li class="detail-side-links"><a href="">{{ professor.name }}</a></li>
+            <li class="detail-side-links"><a href='#' onclick = "professorSelect('{{professor.name}}')">{{ professor.name }}</a></li>
```

Comment:
href seems like it is refreshing the page. do we need this? Can't we change the css styling to look like it is a link?

Reply:
href='#' stops the page from refreshing but also keeps it as a link. Is it refreshing on your browser?